larray/core/session.py

```diff
@@ -1170,14 +1273,18 @@ def display(k, v):
             else:
                 return tmpl(k, v)
 
-        return '\n'.join(display(k, v) for k, v in self.items())
+        res = ''
+        if len(self.meta) > 0:
+            res = '\n'.join(display_metadata(k, v) for k, v in self.meta.items()) + '\n'
+        res += '\n'.join(display_items(k, v) for k, v in self.items())
```

Comment:
then here, something like:
```python
if self.meta:
    res += display_items('Metadata', v) + '\n'
res += '\n'.join(display_items(k, v) for k, v in self.items())
```

The goal being that users can produce something like the following if they want to (maybe it should be the default):
```python
Metadata: 
    title: my title
    author: John Smith
axis1: a ['a0' 'a1' 'a2'] (3)
group1 -> a01: a['a0', 'a1'] (2)
```
**Now, I am unsure it's worth it** to change what you have but having no way to distinguish a metadata item from a "normal" item clearly bothers me.